scripts/saveImageDay.sh

```diff
@@ -69,33 +71,14 @@ fi
 # If upload is true, optionally create a smaller version of the image; either way, upload it
 if [ "$UPLOAD_IMG" = true ] ; then
 	if [[ "$RESIZE_UPLOADS" == "true" ]]; then
-		echo -e "$ME: Resizing '$IMAGE_TO_USE' for uploading"
-		echo -e "$ME: Resizing '$IMAGE_TO_USE' for uploading\n" >> log.txt
-
 		# Create a smaller version for upload
 		convert "$IMAGE_TO_USE" -resize "$RESIZE_UPLOADS_SIZE" -gravity East -chop 2x0 "$IMAGE_TO_USE"
```

Comment:
Here too. Also, are we sure that a convert failure won't somehow overwrite the file?

Reply:
Don't know if convert failure will overwrite the file, but it was like this in the previous version.  I won't be able to test it for a couple weeks.

Reply:
Fair enough.